argschema/fields/__init__.py

```diff
@@ -1,6 +1,8 @@
 '''sub-module for custom marshmallow fields of general utility'''
+from marshmallow.fields import *
```

Comment:
Yes, agreed.  I think my only other concern is that you can't distinguish marshmallow core fields from argschema additions this way.  Do you think that's worth worrying about?